tests/Unit/API/Google/AdsCampaignTest.php

```diff
@@ -66,30 +71,14 @@ public function test_get_campaigns() {
 				'campaign_id'         => self::TEST_CAMPAIGN_ID,
 				'geo_target_constant' => 'geoTargetConstants/2158',
 			],
-			[
-				'campaign_id'         => self::TEST_CAMPAIGN_ID,
-				'geo_target_constant' => '',
-			],
-			[
-				'campaign_id'         => self::TEST_CAMPAIGN_ID,
-				'geo_target_constant' => null,
-			],
```

Comment:
I can see these aren't relevant anymore because they will be skipped in a query. But what about other invalid locations that we expect to skip or throw an exception for:

```
geoTargetConstants/123456789
foobar/2158
```

Reply:
Thanks for it, added a test case for it in c7436c8.